aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java

```diff
@@ -217,17 +212,6 @@ public synchronized void initialize(
                 databaseConnectionHandle = sqliteStorageHelper.getWritableDatabase();
                 this.sqlCommandFactory = new SQLiteCommandFactory(modelSchemaRegistry, databaseConnectionHandle);
 
-                /*
-                 * Create INSERT INTO TABLE_NAME statements for all SQL tables
-                 * and compile them and store in an in-memory map. Later, when a
-                 * {@link #save(T, Consumer, Consumer)} operation needs to insert
-                 * an object (sql rows) into the database, it can bind the input
-                 * values with the prepared insert statement.
-                 *
-                 * This is done to improve performance of database write operations.
-                 */
-                this.insertSqlPreparedStatements = getInsertSqlPreparedStatements();
```

Comment:
Can you delete the `getInsertSqlPreparedStatements()` method itself now too?

Reply:
Ah yes good call!